pebblo/app/api/api.py

```diff
@@ -48,3 +49,9 @@ def promptgov(data: dict):
         prompt_obj = PromptGov(data=data)
         response = prompt_obj.process_request()
         return response
+
+    @staticmethod
+    def classify_data(data: dict):
```

Comment:
Can you add input structure

Reply:
Let's take care of this in or post https://github.com/daxa-ai/pebblo/pull/530 to avoid conflict with that PR.